vividus-plugin-browserstack/src/main/java/org/vividus/selenium/browserstack/BrowserStackCapabilitiesConfigurer.java

```diff
@@ -20,26 +20,49 @@
 import org.vividus.bdd.context.IBddRunContext;
 import org.vividus.selenium.AbstractDesiredCapabilitiesConfigurer;
 
-public class BrowserStackCapabilitiesConfigurer extends AbstractDesiredCapabilitiesConfigurer
+public class BrowserStackCapabilitiesConfigurer extends AbstractDesiredCapabilitiesConfigurer<BrowserStackLocalOptions>
 {
+    private static final String BSTACK_KEY = "bstack:options";
+
     private boolean browserStackEnabled;
+    private String skipUrlsPattern;
 
-    public BrowserStackCapabilitiesConfigurer(IBddRunContext bddRunContext)
+    public BrowserStackCapabilitiesConfigurer(IBddRunContext bddRunContext,
+            BrowserStackLocalManager browserStackLocalManager)
     {
-        super(bddRunContext);
+        super(bddRunContext, browserStackLocalManager);
     }
 
     @Override
     public void configure(DesiredCapabilities desiredCapabilities)
     {
         if (browserStackEnabled)
         {
-            configureTestName(desiredCapabilities, "bstack:options", "sessionName");
+            configureTunnel(desiredCapabilities, tunnelId ->
+            {
+                putNestedCapability(desiredCapabilities, BSTACK_KEY, "local", true);
```

Comment:
is it required? isn't `localIdentifier ` enough?

Reply:
yes, both local and localId are required